api/provider/logger/noteMsg.go

```diff
@@ -21,55 +21,55 @@ const (
 // in green colour
 func (logger *Logger) Info(message ...string) {
 
-    outputMsg := strings.Join(message, " ")
+	outputMsg := strings.Join(message, " ")
 
 	if logger.Verbose {
```

Comment:
Not sure! here logger func should not be checking the verbose condition. Instead I think(ready for discussion) logger func should have a separte mode called verbose and on having that active the info/any program should Print. Not using if else condition is what I mean

Reply:
Okay will have a discussion on that 👍